singlestoredb/management/manager.py

```diff
@@ -148,7 +148,7 @@ def _get(self, path: str, *args: Any, **kwargs: Any) -> requests.Response:
 
         """
         if self._params:
-            kwargs['params'] = self._params
```

Comment:
I think we might want to create a new dict each time. Doing it this way will modify the params dict that is passed in, which may have other side-effects. The same is true of other instances.